javascript/javascript2/week1/class79excercise.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>myFavouriteDish</title>
+</head>
+<body>
+    
+
+    <script>
+       const favouriteDishes =["bread", "pizza", "lasagne"];
+       const body = document.querySelector("body")
+    console.log(favouriteDishes);
+    const ul = document.createElement("ul");
+    console.log(ul);
+    ul.appendChild();
+    
+   for (let i=0; i<favouriteDishes.length;i++){
+       const liTag =document.createElement("li");
+       liTag.innerHTML = favouriteDishes[i]
+       ulTag.appendChild(liTag)       
+   }
+    
```

Comment:
Almost!
here:

```
      const favouriteDishes =["bread", "pizza", "lasagne"];
      const body = document.querySelector("body")
      console.log(favouriteDishes);
      const ul = document.createElement("ul");
      console.log(ul);
      body.appendChild(ul);

      for (let i=0; i<favouriteDishes.length;i++){
        const liTag =document.createElement("li");
        liTag.innerHTML = favouriteDishes[i]
        ul.appendChild(liTag)       
      }
```

Reply:
I have changed it to these your code snippet, its working great now,.,Thanks